sass/core/_parser.scss

```diff
@@ -27,6 +27,12 @@
   $do: null;
   $value: $key;
 
+  // Throw an error if the missing key flag is not configured correctly.
+  @if not index(('error', 'warn', 'null', 'silent'), $handle-missing-keys) {
```

Comment:
This only allows `'null'` as a string value. I was expecting it to be an actual `null`. We could support both, or we should clearly document which one we mean. I'd be ok with either, but think `null` has to be valid without the quotes. (from a quick test, I think adding unquoted null to the list is all we need for a fix here)

Reply:
Ah, yes. I assumed a string value would make more sense for configuration, as opposed to an actual null value since that tends to have secondary effects in SASS. Supporting both would a safe bet, I'll update the list to also include an actual `null` value.

Reply:
Updated the configuration to also include `null`. :)
Thanks for looking into this, appreciate the time.

Reply:
@jcamilleri13 I can merge and make this change myself if that's faster - just wondered if you had thoughts on it.